### PR TITLE
ddl, sessionctx: update interfaces and comments

### PR DESCRIPTION
**Pull request by zimulala**
## What have you changed? (mandatory)
The PR(#6449) only adjusts the structure of ddl for executing DDL parallel. In that PR some comments need to modify the original code, in order to reduce the complexity, so I put this updating to this PR.
The main changes are as follows:

- replace 'workerVar' with 'binlogCli' (by comments in #6449)

- updating interfaces

- updating comments (by comments in #6449)

- updating logs

## What are the type of the changes (mandatory)?

- Improvement 

## How has this PR been tested (mandatory)?

Update a test.

## Does this PR affect documentation (docs/docs-cn) update? (optional)

No

## Refer to a related PR or issue link (optional)

#6449

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @shenli

File: ddl/ddl.go [hunk 1]
```diff
@@ -185,18 +185,18 @@ type DDL interface {
 	// OwnerManager gets the owner manager, and it's used for testing.
 	OwnerManager() owner.Manager
 
-	// WorkerVars gets the session variables for DDL worker. It's exported for testing.
-	WorkerVars() *variable.SessionVars
-	// SetHook sets the hook. It's exported for testing.
+	/*
+	   Export the following interfaces for testing.
```
Comment (coocood) [inline ddl/ddl.go]: We can define those methods in `_test.go` files.

**Comment by zimulala** [comment]
PTAL @coocood @winkyao  @shenli

**Comment by zimulala** [comment]
PTAL @shenli @coocood @ciscoxll

**Comment by shenli** [comment]
LGTM

**Comment by ciscoxll** [comment]
@zimulala Please fix the `run-common-test` and `run-integration-common-test`.

**Comment by zimulala** [comment]
PTAL @ciscoxll

**Pull request merged**